test/studies/prk/stencil/optimized/SKIPIF

```diff
@@ -0,0 +1,2 @@
+CHPL_TEST_PERF != on
+CHPL_TEST_PERF_LABEL != ml-
```

Comment:
Any reason to not run this as a correctness and/or a single locale performance test as well?